seedDB/README.md

```diff
@@ -1,25 +1,7 @@
-Files to seed a new cBioPortal database.
+These files are MySQL database dump for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with release **1.3.1** of cBioPortal.  
```

Comment:
and db.version in https://github.com/cBioPortal/cbioportal/blob/v1.3.1/pom.xml is also 1.3.0, so this is good.